CONTRIBUTING.md

```diff
@@ -1,107 +1,111 @@
-
-```
-GITHUB_ID=<client_id>
-GITHUB_SECRET=<client_secret>
-```
-
-### 4. Setup uploadthing for file uploads
-
-Go to [uploadthing.com](https://uploadthing.com/dashboard) and create a project, then copy in the secrets into your `.env`.
-
-### 5. Install dependencies
-
-Use `pnpm` to install dependencies.
-
-```
-pnpm install
-```
-
-### 6. Push Database Schema and Seed
-
-```
-turbo db:push
-turbo db:seed
-```
-
-### Running the dev server
-
-Finally, you can run the dev server:
-
-```
-pnpm dev
-```
-
-### Sync data
-
-Once the server is running you can seed and sync data
-
-1. Navigate to [http://localhost:5173/api/auth/github](http://localhost:5173/api/auth/github) to authenticate with GitHub OAuth.
-
-#### Tips
-
-if you ever need to blow away everything in your data and reseed run:
-
-```
-pnpm refresh
-```
-
-[planetscale-quick-start]: https://planetscale.com/docs/tutorials/planetscale-quick-start-guide
-[new-oauth]: https://github.com/settings/applications/new
-
-#### Dependencies
-
-UI Library: [shadcn](https://ui.shadcn.com/)
+# Contributing
+
+Thank you for considering contributing to TypeHero. As an open sourced project
+we only exist by contributions from users like you.
+
+To contribute you will first need to fork the repo. To get the repository up and
+running, please look at [LOCAL.md](/LOCAL.MD) for setup instructions.
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the
+[CONTRIBUTING.md Code of Conduct](blob/master/CODE_OF_CONDUCT.md). By
+participating you are expected to uphold this code. Please report unacceptable
+behavior to <placeholder@typehero.dev>
+
+## I Want to Contribute
+
+> ### Legal Notice
+>
+> When contributing to this project, you must agree that you have authored 100%
+> of the content, that you have the necessary rights to the content and that the
+> content you contribute may be provided under the project's license.
+
+### Reporting Bugs
+
+#### Before Submitting a Bug Report
+
+A good bug report shouldn't leave others needing to chase you up for more
+information. Therefore, we ask you to investigate carefully, collect
+information, and describe the issue in detail _in your report_. Please complete
+the following steps in advance to help us fix any potential bug as fast as
+possible:
+
+1. Document the page(s) the bug is happening on.
+2. Take applicable Screenshots if necessary, and create any relevant stack
+   traces (if possible)
+3. If you are experiencing this bug on your local, ensure that it is not do to
+   an incorrect local configuration.
+4. Check if a similar issue already exists in the
+   [bug tracker](issues?q=label%3A:bug)
+
+#### How do I Submit a Good Bug Report?
+
+> ### You must never report security related issues, vulnerabilities, or bugs including sensitive information to the issue tracker, or elsewhere in public. Instead, sensitive bugs must be sent by email to [bugs.sensitive@typehero.dev](mailto:bugs.sensitive@typehero.dev)
```

Comment:
@bautistaaa we need a proper email for this.

@d0nutptr Are there free email options for a custom domain out there?

Reply:
In theory Zoho Mail has a free tier?

Reply:
could we create email aliases on cloudflare guys? we can then forward them to a single ingress email say "security@typehero.dev" 

example, ```
sus@typehero.dev (alias) --> security@typehero.dev
```

> tbh, it kinda defeats the point of having different aliases since one would assume to have different inboxes)

but one can also forward first to a worker & assign some extra info depending on the alias context & then forward to the security email. we can also handle them however we like.

example, ```
sus@typehero.dev (alias) --> cloudflare worker --> security@typehero.dev
```

Reply:
Cloudflare??? who buddy calm down, we're a Vercel company :p

Reply:
> Cloudflare??? whoa buddy calm down, we're a Vercel company :p

my time to shine

![image](https://github.com/bautistaaa/typehero/assets/36449128/5d3f9e4c-8aa5-44c8-9016-ae4228eaff24)

Reply:
damn i booby'd right into this trap.